code/modules/antagonists/devil/true_devil/inventory.dm

```diff
@@ -1,4 +1,4 @@
-/mob/living/carbon/true_devil/doUnEquip(obj/item/I, force, newloc, no_move, invdrop = TRUE, was_thrown = FALSE)
+/mob/living/carbon/true_devil/doUnEquip(obj/item/I, force, newloc, no_move, invdrop = TRUE, was_thrown = FALSE, bypass_strip_delay_other = TRUE) //NSV13
```

Comment:
I suggest using a shorter variable name like `bypass_delay` or similar for readability.

Reply:
done